pkg/scheduler/core/generic_scheduler.go

```diff
@@ -486,6 +486,17 @@ func (g *genericScheduler) findNodesThatFit(pod *v1.Pod, nodes []*v1.Node) ([]*v
 
 		checkNode := func(i int) {
 			nodeName := g.cache.NodeTree().Next()
+
+			// Iterate each plugin to verify current node
+			status := g.framework.RunFilterPlugins(pluginContext, pod, nodeName)
+			if !status.IsSuccess() {
```

Comment:
Similar to the code block at line 508, we should check if the returned status code is an `Error` and in that case, we should:
```
errs[status.Message()]++
```

Reply:
Done.